src/StructuredLogger/BinaryLogger/BuildSubmissionStartedEvent.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Build.Framework;
+
+namespace StructuredLogger.BinaryLogger
+{
+    internal class BuildSubmissionStartedEvent() : BuildStatusEventArgs("", null, null, DateTime.UtcNow)

```

Comment:
it's good to specify argument names for literals such as strings, null, true and false, so when you read the code you know what parameter they map to